tests/tt_eager/python_api_testing/unit_testing/test_softmax.py

```diff
@@ -19,151 +19,97 @@
 from models.utility_functions import torch2tt_tensor, tt2torch_tensor, pad_by_zero
 
 
-@skip_for_wormhole_b0()
-@pytest.mark.parametrize("inplace", [True, False])
-def test_softmax(device, inplace):
-    torch.manual_seed(0)
-    sm_op = ttl.operations.primary.softmax_in_place if inplace else ttl.tensor.softmax
+# @skip_for_wormhole_b0()
```

Comment:
Did you mean to comment out the tests?

If no longer need this code, we should just delete it